nvflare/job_config/fed_job.py

```diff
@@ -14,22 +14,30 @@
 import os.path
 import re
 import uuid
-from typing import Any, List, Union
+from typing import Any, List, Optional, Union
 
 from nvflare.apis.executor import Executor
-from nvflare.apis.filter import Filter
+from nvflare.apis.filter import Filter, FilterType
 from nvflare.apis.impl.controller import Controller
 from nvflare.apis.job_def import ALL_SITES, SERVER_SITE_NAME
-from nvflare.app_common.executors.script_executor import ScriptExecutor
+from nvflare.app_common.executors.client_api_launcher_executor import ClientAPILauncherExecutor
+from nvflare.app_common.executors.in_process_client_api_executor import InProcessClientAPIExecutor
```

Comment:
We need serious rework on this. This long list of imports implies that something is wrong with the design. We cannot possibly take all the details of each special object into this framework level code!

Reply:
removed from this pr, will refactor fed_job.py to improve layering